bin/node/src/aleph_cli.rs

```diff
@@ -37,4 +40,18 @@ impl AlephCli {
     pub fn backup_path(&self) -> Option<PathBuf> {
         self.backup_path.clone()
     }
+
+    pub fn insert_default_backup_path(&mut self, config: &Configuration) {
+        if !self.no_backup {
+            let path = self.backup_path.get_or_insert(
+                config
+                    .base_path
+                    .as_ref()
+                    .expect("Please specify base path")
+                    .path()
+                    .join(DEFAULT_BACKUP_FOLDER),
+            );
+            eprintln!("No backup path provided, using default path: {:?} for AlephBFT backups. Please do not remove this folder", path);
```

Comment:
This will be printed also when `backup_path` was `Some` at the beginning (that means the user DID actually provide backup path)

Reply:
Yep, thanks for noticing. I tried to do something smart